packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/recent.tsx

```diff
@@ -0,0 +1,109 @@
+import classnames from 'classnames';
+import React from 'react';
+import { Icon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import RecentWorkspace from './recent-workspace';
+import RootStore from 'Stores/index';
+import SaveModal from './save-modal';
+import DeleteDialog from './delete-dialog';
+import { getSavedWorkspaces } from '@deriv/bot-skeleton';
+import './index.scss';
+
+type TRecentComponent = {
+    is_explanation_expand: boolean;
+    recent_strategies: [];
+    toggleExplanationExpand: boolean;
+    is_strategy_removed: boolean;
+    handleFileChange: () => void;
+    toggleStrategies: (param: boolean) => void;
+    setRecentStrategies: () => void;
+};
+
+const RecentComponent = ({
+    is_explanation_expand,
+    recent_strategies,
+    toggleExplanationExpand,
+    toggleStrategies,
+}: TRecentComponent) => {
+    const [getdata, setGetdata] = React.useState([recent_strategies]);
+
+    React.useEffect(() => {
+        toggleStrategies(true);
+        getSavedWorkspaces().then(data => {
+            setGetdata(data);
+        });
+    }, []);
+
+    if (getdata?.length) {
+        return (
+            <div className='load-strategy__container load-strategy__container--has-footer'>
+                <div className='load-strategy__recent'>
+                    <div className='load-strategy__recent-files'>
+                        <div className='load-strategy__title'>
+                            <Localize i18n_default_text='Your Bots' />
+                        </div>
+                        <div className='load-strategy__recent-files-list'>
+                            {recent_strategies.map((workspace, index) => (
+                                <RecentWorkspace key={workspace.id} workspace={workspace} index={index} />
+                            ))}
+                        </div>
+                        {/* <SaveModal />
+                        <DeleteDialog /> */}
+                    </div>
+                </div>
+            </div>
+        );
+    }
+
+    return (
+        <div className='load-strategy__container'>
+            <div className='load-strategy__recent-empty'>
+                <Icon icon='IcEmptyFolder' className='load-strategy__recent-empty-icon' size={128} />
+                <div className='load-strategy__recent-empty-title'>
```

Comment:
I cannot find load-strategy__recent-empty-title class. ... 
one more thing is that can we use <Text> instead? (also for the rest)

Reply:
this is a legacy change will fix it in improvement PR because the older load model css is breaking